autogpt/commands.py

```diff
@@ -122,15 +122,16 @@ def get_datetime():
 
 def google_search(query, num_results=8):
     search_results = []
-    for j in ddg(query, max_results=num_results):
+    for j in search(query, num_results=num_results):
```

Comment:
Why does this PR try to sneak in a change away from using Duck Duck Go?

The PR that changed that allowed us to start getting much better results and less errors.